server/adapters/groups.js

```diff
@@ -0,0 +1,32 @@
+var Sequelize = null;
```

Comment:
Hmm, I was very afraid of Javascript multiple file/module inclusions, but after some researches I found that you're right! Node actually performing caching on the modules it loads, with `require()` it's okay to do that. https://nodejs.org/docs/latest/api/modules.html#modules_caching